src/tracing/CMakeLists.txt

```diff
@@ -43,9 +43,9 @@ endfunction()
 
 set(osd_traces oprequest.tp osd.tp pg.tp)
 add_tracing_library(osd_tp "${osd_traces}" 1.0.0)
-add_tracing_library(rados_tp librados.tp 2.0.0)
+add_tracing_library(librados_tp librados.tp 2.0.0)
```

Comment:
@tchaikov Hello, here are some question I'm not sure:
- This fix works because this line? - https://github.com/ceph/ceph/blob/master/src/tracing/CMakeLists.txt#L39:   
```
  set_target_properties(${name} PROPERTIES
  OUTPUT_NAME ${name}
```     
  See [set_target_properties](https://cmake.org/cmake/help/v3.0/command/set_target_properties.html)
- it's weird that I can not find ibrados_tp library (the same for other *_tp libraries). According to   
```
install(TARGETS rados_tp osd_tp rbd_tp os_tp DESTINATION ${CMAKE_INSTALL_LIBDIR})
```
it should be here:
```
ls -1 build/lib/*rados*
build/lib/librados.so
build/lib/librados.so.2
build/lib/librados.so.2.0.0
```